custom_components/tuya_local/devices/solar_inverter_gtb_series.yaml

```diff
@@ -0,0 +1,161 @@
+name: Smart Inverter GTB Series
+products:
+  - id: bf27badb1abd9bfc5beeuq
+    name: GTB Series
+primary_entity:
+  entity: sensor
+  class: energy
+  dps:
+    - id: 2
+      type: integer
+      name: sensor
+      unit: kWh
+      class: total_increasing
+      mapping:
+        - scale: 100
+    - id: 15
+      name: model
+      type: string
+    - id: 16
+      name: inverter_id
+      type: string
+secondary_entities:
+  - entity: sensor
+    class: power
+    category: diagnostic
+    dps:
+      - id: 10
+        type: integer
+        name: sensor
+        unit: W
+        force: true
+        class: measurement
+        mapping:
+          - scale: 1
+  - entity: sensor
+    class: temperature
+    category: diagnostic
+    dps:
+      - id: 18
+        type: integer
+        name: sensor
+        unit: C
+        force: true
+        class: measurement
+        mapping:
+          - scale: 10
+  - entity: sensor
+    name: DC voltage
+    class: voltage
+    category: diagnostic
+    dps:
+      - id: 3
+        type: base64
+        name: sensor
+        unit: V
+        optional: true
+        persist: true
```

Comment:
persisting is default behaviour. Only use the persist option if you want to set it to false.